### PR TITLE
[JAMES-3226] drops unused ui-bundle

### PR DESCRIPTION
**Pull request by jeantil**
It seems that an effort to use an customized UI bundle was started and abandonned halfway. The bundle currently isn't used in the website generation and can confuse contributors. 

I propose to remove them, since the modifications it contain can now be replicated by using antora supplementary files to override the default UI

### CODE REVIEW

**Review by chibenwa — COMMENTED** [review]
Shouldn't we drop `/antora-ui` too?

**Review by chibenwa — APPROVED** [review]

**Comment by jeantil** [comment]
I think we should keep antora-ui for now I am experimenting with different schemes for modifying the UI I'll do a followup cleanup PR once I have something working in a satisfying way

**Comment by jeantil** [comment]
@chibenwa I updated this PR with a couple commits demonstrating how to customize UI, the demo is a proof of concept I didn't spend too much time on styling the icons for example, some of the links are not perfect but it shows how it's done :)

File: doc-sites/package-lock.json [hunk 1]
```diff
@@ -5,29 +5,37 @@
   "requires": true,
```
Comment (chibenwa) [inline doc-sites/package-lock.json:5]: I'm js naive but shouldn't package.json be ignored?

**Review by chibenwa — COMMENTED** [review]
Can you share a screenshot?

File: doc-sites/package-lock.json (hunk 1 above)
Comment (jeantil) [reply doc-sites/package-lock.json:5]: According to the [documentation](https://docs.npmjs.com/cli/v7/configuring-npm/package-lock-json)

> package-lock.json is automatically generated for any operations where npm modifies either the node_modules tree, 
 or package.json. It describes the exact tree that was generated, such that subsequent installs are able to generate identical trees, regardless of intermediate dependency updates.


So we do need to commit both package.json and package.lock.json

**Comment by jeantil** [comment]
Screenshot from 2021-04-03 22-07-37
Screenshot from 2021-04-03 22-07-34

**Comment by jeantil** [comment]
Screenshot from 2021-04-03 22-15-00

**Review by chibenwa — APPROVED** [review]

**Comment by jeantil** [comment]
ideally I would prefer to have this merged before merging to live and hard resetting staging :)

**Comment by chibenwa** [comment]
Yes, I did not want to hurry too ;uch to give a chance for over people to review this.

I am unsure how many people are actively watching this repository.

I will merge it now, it is several days already.

**Pull request merged**